### PR TITLE
Fix tour

### PR DESCRIPTION
**Pull request by eugene70**

### CODE REVIEW

**Comment by eugene70** [comment]
need to rebuild, how?

**Comment by eugene70** [comment]
@byeonggukgong, @eed3si9n, please take a look.

**Comment by eed3si9n** [comment]
I'd love to review, but I can't myself. Maybe find someone who can? /cc @yoohaemin

**Comment by byeonggukgong** [comment]
@eugene70 @eed3si9n I'll review it this weekend, if I can.

**Comment by SethTisue** [comment]
@byeonggukgong ping

**Comment by byeonggukgong** [comment]
@SethTisue Ooops, I was so busy lately that I forgot. Thanks for the ping. Maybe this weekend I can review this translated documents. :)

File: _ko/tour/unified-types.md [hunk 1]
```diff
@@ -8,38 +8,74 @@ language: ko
...
-
-    This is a string
-    732
-    c
-    true
-    <function>
-    
-윤창석, 이한욱 옮김
\ No newline at end of file
+스칼라에서는 숫자 값과 함수를 포함한 모든 값이 타입을 가지고 있습니다. 다음의 도표는 타입 계층구조를 보여줍니다.
```
Comment (byeonggukgong) [inline _ko/tour/unified-types.md]: 타입 계층구조
타입 계층 구조

On Wikipedia, I found 계층구조 as 계층 구조. It's up to the author to choose.

File: _ko/tour/unified-types.md [hunk 2]
```diff
@@ -8,38 +8,74 @@ language: ko
...
+
+<a href="{{ site.baseurl }}/resources/images/tour/unified-types-diagram.svg"><img  style="width:100%" src="{{ site.baseurl }}/resources/images/tour/unified-types-diagram.svg" alt="스칼라 타입 계층구조"></a>
+
+
+## 스칼라 타입 계층구조 ##
+
+[`Any`](https://www.scala-lang.org/api/2.12.1/scala/Any.html)는 모든 타입들의 슈퍼타입이며 탑타입이라고도 합니다. `Any`에는 `equals`, `hashCode`, `toString` 같은 특정 범용 메서드가 정의되어 있으며 직접적으로 두개의 서브클래스: `AnyVal`과 `AnyRef`를 가지고 있습니다.
```
Comment (byeonggukgong) [inline _ko/tour/unified-types.md]: 슈퍼타입, 탑타입

It's necessary to discuss whether to translate the pronunciation.

Comment (byeonggukgong) [inline _ko/tour/unified-types.md]: 두개의
두 개의 or 2개의

I know this grammer is correct.

File: _ko/tour/unified-types.md [hunk 3]
```diff
@@ -8,38 +8,74 @@ language: ko
...
+
+`AnyVal`은 값 타입을 대표합니다. `Double`, `Float`, `Long`, `Int`, `Short`, `Byte`, `Char`, `Unit`, `Boolean`의 미리 정의된 아홉개의 값 타입이 있으며 이 타입들은 널 값을 가질 수 없습니다. `Unit`은 의미 없는 정보를 갖는 값 타입입니다. `()`와 같이 문자 그대로 선언 할 수있는`Unit`의 인스턴스는 오직 하나만 있습니다. 모든 함수는 무언가를 반환해야하기 때문에 때때로 `Unit`은 유용한 반환 타입입니다.
```
Comment (byeonggukgong) [inline _ko/tour/unified-types.md]: 아홉개의
아홉 개의 or 9개의

I know this grammer is correct.

File: _ko/tour/unified-types.md [hunk 4]
```diff
@@ -8,38 +8,74 @@ language: ko
...
+
+`AnyRef`는 참조 타입을 대표합니다. 값 타입이 아닌 모든 타입은 참조 타입으로 정의됩니다. 스칼라에서 모든 사용자정의 타입은 `AnyRef`의 서브타입입니다. 스칼라가 자바 실행 환경에서 사용된다면 `AnyRef`는 `java.lang.Object`에 해당합니다.
```
Comment (byeonggukgong) [inline _ko/tour/unified-types.md]: 사용자정의 타입은
사용자 정의 타입은

I know this grammer is correct.

File: _ko/tour/classes.md [hunk 5]
```diff
@@ -8,41 +8,99 @@ language: ko
 
 next-page: traits
 previous-page: unified-types
+topics: classes
+prerequisite-knowledge: no-return-keyword, type-declaration-syntax, string-interpolation, procedures
 ---
 
-스칼라의 클래스는 런타임에 많은 객체로 인스턴스화 될 수 있는 정적 템플릿이다.
-아래는 'Point' 클래스의 정의이다.
+스칼라의 클래스는 객체를 만들기위한 설계도입니다. 클래스에는 _멤버_ 라고 통칭할 수 있는 메서드, 값, 변수, 타입, 객체, 트레잇, 클래스를 포함할 수 있습니다. 타입, 객체, 트래잇은 투어에서 나중에 다루겠습니다.
```
Comment (byeonggukgong) [inline _ko/tour/classes.md]: 만들기위한
만들기 위한

트래잇은 투어에서 나중에
트레잇은 투어에서 나중에

I know this grammer is correct.

File: _ko/tour/classes.md [hunk 6]
```diff
@@ -8,41 +8,99 @@ language: ko
...
+}
 
-이 프로그램은 `main` 메서드를 가지고 있는 최상위 싱글톤의 형태로 실행가능한 어플리케이션 클래스를 정의한다. `main` 메서드는 새로운 `Point`를 생성하고 변수 `pt`에 저장한다. `val` 키워드로 정의된 값은 변경을 허용하지 않는다는 점에서 `var` 키워드로 정의된 값(`Point` 클래스 참)과는 다르다는 점에 주의하자. 즉, 이 값은 상수이다.
+val point1 = new Point(2, 3)
+point1.x  // 2
+println(point1)  // prints (2, 3)
+```
 
-이 프로그램의 결과는 아래와 같다:
+이 `Point` 클래스에는 변수`x`, `y`와 메서드 `move`, `toString`의 네 멤버가 있습니다. 다른 많은 언어와 달리 기본 생성자는 클래스 서명부`(var x : Int, var y : Int)`에 있습니다. `move` 메소드는 두 개의 정수 인자를 취하여 정보를 전달하지 않는 Unit 값 `()`을 리턴합니다. 이것은 자바와 유사한 언어에서 'void'와 대략 일치합니다. 반면에 `toString`은 인자를 취하지 않고 `String` 값을 반환합니다. `toString`은 [`AnyRef`](unified-types.html)의 `toString`을 대체하므로 `override` 예약어로 지정됩니다.
```
Comment (byeonggukgong) [inline _ko/tour/classes.md]: 다른 많은 언어와 달리
많은 다른 언어와 달리

Translated like this, it's more natural.

Comment (byeonggukgong) [inline _ko/tour/classes.md]: 클래스 서명부

It need more information about 'class signature'. Then we can translate it more naturally.

File: _ko/tour/unified-types.md (hunk 1 above)
Comment (eugene70) [reply _ko/tour/unified-types.md]: I think '계층구조' is better.

File: _ko/tour/unified-types.md (hunk 2 above)
Comment (eugene70) [reply _ko/tour/unified-types.md]: I found,

'슈퍼' is right: 
https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=120147

'톱' is right:
https://www.korean.go.kr/front/mcfaq/mcfaqView.do?&mn_id=62&mcfaq_seq=946&pageIndex=1

File: _ko/tour/unified-types.md (hunk 4 above)
Comment (eugene70) [reply _ko/tour/unified-types.md]: You are right.

File: _ko/tour/classes.md (hunk 6 above)
Comment (eugene70) [reply _ko/tour/classes.md]: How about '클래스 서명부(signature)' or '클래스 시그니처'?

**Comment by eugene70** [comment]
Take a look again.

**Comment by eed3si9n** [comment]
I am happy to merge this as is assuming you've addressed the review comments. We can always come back and fix things up later.

**Comment by eugene70** [comment]
Thanks @eed3si9n & @byeonggukgong !
I will go on.

**Pull request merged**